index.js

```diff
@@ -0,0 +1,94 @@
+let isGaming = false;
+let answerNumber = "";
+
+function inputNumber() {
+  let gameInput = document.getElementById("gameInput").value;
+  document.getElementById("gameInput").value = "";
+  if (!isGaming) {
+    if (gameInput == "1") {
+      gameStart();
+    } else if (gameInput == "9") {
+      gameStop();
+    } else {
+      alert("1 또는 9를 입력하세요");
+    }
+  } else {
+    if (gameInput.length == 3) {
+      checkNum(gameInput);
+    } else if (gameInput == 9) {
+      gameStop();
+    } else {
+      alert("3자리 숫자를 입력해주세요");
+    }
+  }
+  
+}
+
+function gameStart() {
+  isGaming = true;
+  document.getElementById("gameTitle").innerHTML =
+    "<div>게임이 진행중입니다. 3자리 숫자를 입력해주세요.</div>";
+  randomNumber();
+}
+
+function gameStop() {
+  isGaming = false;
+  answerNumber = "";
+  document.getElementById("gameTitle").innerHTML =
+    "<div>게임을 새로 시작하려면 1, 종료하려면 9를 입력하세요</div>";
+  document.getElementById("gameResult").innerHTML = "";
+}
+
+//랜덤 3자리 숫자를 뽑아주는 함수
+function randomNumber() {
```

Comment:
함수명은 동사로 시작하면 좋을 것 같습니다.

```suggestion
function generateRandomNumber() {
```